bigquery_etl/parse_metadata.py

```diff
@@ -0,0 +1,78 @@
+"""
+Parsing of metadata yaml files.
+"""
+
+import re
+import yaml
+
+
+class Metadata:
```

Comment:
Accessing metadata is used in several places. Having a separate class with all the logic for parsing the metadata files reduced code duplications